neuraxle/steps/util.py

```diff
@@ -517,7 +511,45 @@ def get_cache_path_for(self, data_input) -> str:
         raise NotImplementedError()
 
 
-class PickleValueCachingWrapper(BaseValueCachingWrapper):
+class OutputTransformerMixin:
+    """
+    Base output transformer step that can modify data inputs, and expected_outputs at the same time.
+    """
```

Comment:
I want to understand: why was this mixin added in this PR? is is used here? Can we merge this before in another PR?